working-group-lifecycle.md

```diff
@@ -0,0 +1,67 @@
+# Working Group Life Cycle
+
+Contributors interested in starting a new working group are encouraged to first review the existing
+working groups and projects. It is very likely that an existing working group could be extended to
+cover the new topic. In that case please bring your time and energy into that existing working
+group. Your contributions are very welcome!
+
+However, if no existing effort covers the topic feel free to make use of the existing working group
+meetings, mail lists, and/or slack channels to solicit support for a new working group.
+
+Please note that running a working group successfully requires an extended effort.
+Be prepared to commit at least several hours per week for a year or more.
+
+The Working Group Life Cycle begins with establishing commitments from other interested
+contributors then making a proposal to the Technical Advisory Committee (TAC). If accepted, the
+Working Group will begin as `Incubating`. This is a boot strapping phase for the group to adopt
+OpenSSF community policies and norms. Once those are met the group becomes `Active` and will be
+fully recognized in OpenSSF. Either upon completing its objectives or no longer making forward
+progress a project will move to `End-of-Life`. The details for each phase follow.
+
+## To become `Incubating`
+
+* Proposal of scope for review by TAC
```

Comment:
I would love if we could provide some "seed" resources to incubating projects and lay out a guide for how to access each one. This is arguably the time when they'll need our support the most. 

What subset of the following could we offer without too much admin overhead?  
"Have access to community resources (Zooms, YouTube channels, GitHub, Slack channels, etc.)"

Reply:
+1! What do you have in mind for these resources?

Reply:
We should probably add one additional state, `proposed`, for a proposed WG that hasn't entered the `incubating` state.

Reply:
> We should probably add one additional state, `proposed`, for a proposed WG that hasn't entered the `incubating` state.

I think I'm fine with just going straight to `incubating`.

> What subset of the following could we offer without too much admin overhead?
"Have access to community resources (Zooms, YouTube channels, GitHub, Slack channels, etc.)"

Now that that's included in the preamble, I don't think we need to include it here. Alternatively, something like "Can use community resources to organize, but do not have dedicated channels"